src/lib.rs

```diff
@@ -1,9 +1,9 @@
-//! An experiment crate for proposals from the error handling project group.
+//! An experimental crate for proposals from the error handling project group.
 //!
 //! This crate currently contains two experiments, each in their own module.
 //!
 //! 1. An alternative to `Box<dyn Error + ...>` that implements `Error`.
-//! 1. An error reporter that wraps an error and handles iterating over sources
+//! 2. An error reporter that wraps an error and handles iterating over sources
 //!    and formatting a full error report.
 //!
 #![feature(try_trait_v2)]
```

Comment:
We should probably enable some extra lints to help us make sure we're setting up some comprehensive documentation. `eyre` has a good set of these enabled, though I encourage you to look at the lints available because there are probably new ones we could enable since it's been a while if I set this up.

https://github.com/yaahc/eyre/blob/master/src/lib.rs#L287-L309